approvals/src/main/java/no/sikt/nva/approvals/persistence/DynamoDbApprovalRepository.java

```diff
@@ -93,6 +100,19 @@ public Optional<Approval> findByIdentifier(NamedIdentifier namedIdentifier) thro
         }
     }
 
+    @Override
+    public List<NamedIdentifier> findIdentifiers(Collection<NamedIdentifier> namedIdentifiers)
+        throws RepositoryException {
+        return namedIdentifiers.isEmpty() ? List.of() : fetchIdentifiers(namedIdentifiers);
+    }
+
+    private static <T> List<List<T>> splitToChunks(List<T> list) {
+        return IntStream.range(0, (list.size() + TRANSACT_WRITE_ITEM_LIMIT - 1) / TRANSACT_WRITE_ITEM_LIMIT)
```

Comment:
BATCH_GET_ITEM_LIMIT maybe? maybe just have one limit both for get and later write if needed